### PR TITLE
Rename Drop Target characteristic to Target

### PR DESCRIPTION
**Pull request by libertymayc**
Rename Drop Target characteristic to Target, add borderColor token

### CODE REVIEW

File: packages/theme/css/characteristics/droptarget.css [hunk 1]
```diff
@@ -1,14 +0,0 @@
-.uitk-light,
```
Comment (joshwooding) [inline packages/theme/css/characteristics/droptarget.css:1]: Wonder if there's a way to get Git to track this rename

File: .changeset/famous-pets-notice.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Rename Drop Target characteristic to Target;
```
Comment (joshwooding) [inline .changeset/famous-pets-notice.md:6]: I like how this changeset was done with renames https://github.com/jpmorganchase/uitk/pull/634/files#diff-84f7136ddb182f738b5cbdd44309e8b70e3e25af94c7913fd6b46b92acb9f627R6

I think it's worth copying it here

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comment

File: .changeset/famous-pets-notice.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Rename Drop Target characteristic to Target
```
Comment (joshwooding) [inline .changeset/famous-pets-notice.md:6]:
```diff
- --uitk-dropTarget-borderStyle
- --uitk-dropTarget-borderStyle-hover
- --uitk-dropTarget-borderStyle-disabled
- --uitk-dropTarget-borderWidth
- --uitk-dropTarget-borderWidth-hover
- --uitk-dropTarget-borderWidth-disabled
- --uitk-dropTarget-cursor-disabled
- --uitk-dropTarget-background-hover
  
+ --uitk-target-background-hover
+ --uitk-target-borderStyle
+ --uitk-target-borderStyle-hover
+ --uitk-target-borderStyle-disabled
+ --uitk-target-borderWidth
+ --uitk-target-borderWidth-hover
+ --uitk-target-borderWidth-disabled
+ --uitk-target-cursor-disabled
```

File: .changeset/famous-pets-notice.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
++ --uitk-target-borderStyle
++ --uitk-target-borderStyle-hover
++ --uitk-target-borderStyle-disabled
++ --uitk-target-borderWidth
++ --uitk-target-borderWidth-hover
++ --uitk-target-borderWidth-disabled
++ --uitk-target-cursor-disabled
++ --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover)
++ --uitk-target-borderStyle-hover: solid
+```
```
Comment (joshwooding) [inline .changeset/famous-pets-notice.md:27]:
```suggestion
```diff
- --uitk-dropTarget-background-hover
- --uitk-dropTarget-borderStyle
- --uitk-dropTarget-borderStyle-hover: dashed;
- --uitk-dropTarget-borderStyle-disabled
- --uitk-dropTarget-borderWidth
- --uitk-dropTarget-borderWidth-hover
- --uitk-dropTarget-borderWidth-disabled
- --uitk-dropTarget-cursor-disabled
+ --uitk-target-background-hover
+ --uitk-target-borderStyle
+ --uitk-target-borderStyle-hover: solid
+ --uitk-target-borderStyle-disabled
+ --uitk-target-borderWidth
+ --uitk-target-borderWidth-hover
+ --uitk-target-borderWidth-disabled
+ --uitk-target-cursor-disabled
+ --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover)
```
```

Just makes the order consistent adds the original valud for boderStyle hover

File: .changeset/famous-pets-notice.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
++ --uitk-target-background-hover
++ --uitk-target-borderStyle
++ --uitk-target-borderStyle-hover: solid
++ --uitk-target-borderStyle-disabled
++ --uitk-target-borderWidth
++ --uitk-target-borderWidth-hover
++ --uitk-target-borderWidth-disabled
++ --uitk-target-cursor-disabled
++ --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover)
++ --uitk-target-borderStyle-hover: solid
```
Comment (joshwooding) [inline .changeset/famous-pets-notice.md:26]: Duplicated :)

Comment (libertymayc) [reply .changeset/famous-pets-notice.md:26]: darn

**Review by joshwooding — APPROVED** [review]

**Pull request merged**